### ISSUE TITLE
Close Icon in the notification message does not receive keyboard focus

### ISSUE THREAD

**Issue by wilsonfurtado2000** [jitsi/jitsi-meet#15436]
### What happened?

Close Icon in the notification message does not receive keyboard focus

### Platform

- [X] Chrome (or Chromium based)
- [ ] Firefox
- [ ] Safari
- [ ] Other desktop browser
- [ ] Android browser
- [ ] iOS browser
- [ ] Electron app
- [ ] Android mobile app
- [ ] iOS mobile app
- [ ] Custom app using a mobile SDK

### Browser / app / sdk version

Browser

### Reproducibility

- [ ] The problem is reproducible on meet.jit.si

**Comment by saghul** [comment]
Fixed in https://github.com/jitsi/jitsi-meet/pull/15435
